src/ontology/foodon-edit.owl

```diff
@@ -19736,6 +19754,150 @@ AnnotationAssertion(dc:date obo:FOODON_00004287 "2022-06-15T11:40:45Z"^^xsd:date
 AnnotationAssertion(rdfs:label obo:FOODON_00004287 "food (unseasoned)"@en)
 SubClassOf(obo:FOODON_00004287 obo:FOODON_00002454)
 
+# Class: obo:FOODON_00004291 (food (cut))
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004291 "A food product that has undergone a process of being cut into smaller pieces, chunks or slices."@en)
+AnnotationAssertion(dc:contributor obo:FOODON_00004291 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dc:date obo:FOODON_00004291 "2022-08-24T11:51:23Z"@en)
+AnnotationAssertion(rdfs:label obo:FOODON_00004291 "food (cut)"@en)
+SubClassOf(obo:FOODON_00004291 obo:FOODON_00002454)
+
+# Class: obo:FOODON_00004292 (feeder cow)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004292 "A cow that is usually sold to feed yards for further weight gain."@en)
+AnnotationAssertion(obo:IAO_0000119 obo:FOODON_00004292 "https://en.wikipedia.org/wiki/Cattle"@en)
+AnnotationAssertion(dc:contributor obo:FOODON_00004292 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dc:contributor obo:FOODON_00004292 <https://orcid.org/0000-0002-1107-9135>)
+AnnotationAssertion(dc:date obo:FOODON_00004292 "2022-08-24T12:12:34Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:FOODON_00004292 "store cow"@en)
+AnnotationAssertion(rdfs:label obo:FOODON_00004292 "feeder cow"@en)
+SubClassOf(obo:FOODON_00004292 obo:NCBITaxon_9913)
+
+# Class: obo:FOODON_00004293 (finisher cow)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004293 "A cow that undergoes a final fattening process prior to sale or slaughter."@en)
+AnnotationAssertion(obo:IAO_0000119 obo:FOODON_00004293 "https://en.wikipedia.org/wiki/Cattle"@en)
+AnnotationAssertion(dc:contributor obo:FOODON_00004293 <https://orcid.org/0000-0002-1107-9135>)
+AnnotationAssertion(dc:date obo:FOODON_00004293 "2022-08-24T12:49:23Z"@en)
+AnnotationAssertion(rdfs:label obo:FOODON_00004293 "finisher cow"@en)
+SubClassOf(obo:FOODON_00004293 obo:NCBITaxon_9913)
+
+# Class: obo:FOODON_00004294 (stocker cow)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004294 "A  cow that has been weaned and is being fed in a way to add physical bulk to its body."@en)
+AnnotationAssertion(dc:contributor obo:FOODON_00004294 <https://orcid.org/0000-0002-1107-9135>)
+AnnotationAssertion(dc:date obo:FOODON_00004294 "2022-08-24T12:53:15Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:FOODON_00004294 "stocker cow"@en)
+SubClassOf(obo:FOODON_00004294 obo:NCBITaxon_9913)
+
+# Class: obo:FOODON_00004295 (weanling cow)
+
+AnnotationAssertion(dc:contributor obo:FOODON_00004295 <https://orcid.org/0000-0002-1107-9135>)
+AnnotationAssertion(dc:date obo:FOODON_00004295 "2022-08-24T13:39:16Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:FOODON_00004295 "A young cow that has been weaned and consumes solid food."@en)
```

Comment:
looks like this was meant to be definition?

Reply:
Sorry, my bad! I've just corrected.